packages/router/router.ts

```diff
@@ -1112,15 +1112,6 @@ export function createRouter(init: RouterInit): Router {
       return;
     }
 
-    // Short circuit if navigation is blocked
-    if (
-      Array.from(state.blockers).some(([_, blocker]) => {
-        return blocker.state === "blocked";
-      })
-    ) {
-      return;
-    }
-
```

Comment:
I think his is leftover form very early.  Our entry points are `popstate` and `router.navigate` so we can just check for blocking there